src/MovieDetailed/MovieComponent.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { detailedMovieApi } from '../api/detailedMovieApi';
+import MovieDetailed from './MovieDetailed';
+import { updateSingleMovieData } from './MovieDetailedActions';
+
+class MovieComponent extends Component {
```

Comment:
yep, it's what I wanted to see for single item view

I can point that naming of functions, props can be improved

Reply:
Movie now